packages/amplify-graphiql-explorer/package.json

```diff
@@ -29,7 +29,7 @@
     "eslint-webpack-plugin": "^3.1.1",
     "file-loader": "^6.2.0",
     "fs-extra": "^10.0.0",
-    "graphiql": "^1.3.2",
+    "graphiql": "1.5.16",
```

Comment:
```suggestion
    "graphiql": "^1.5.16", ?
```

Reply:
is question mark a typo? otherwise, I believe we can make this change